### PR TITLE
We don't currently support streaming lexers so document that more carefully

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: doc/src/lexing.md [hunk 1]
```diff
@@ -8,13 +8,8 @@ starting at index 7 to index 10"). There is also a simple mechanism to
...
-This API allows users to deal with streaming data since the parser asks the
-`Lexer` for one token at a time. However, note that users can later ask the
-`Lexer` to return the string from the input matching a lexeme: users need to
-buffer input to provide this information.
-
-Hand-written lexers are not particularly difficult to write and, for better or
-worse, are necessary for many real-world languages. However, a subset of
-languages can use a simpler `lex`/`flex` style approach to lexing, for which
-[`lrlex`](lrlex.html) can be used.
+A a subset of languages can use a simple `lex`/`flex` style approach to lexing,
```
Comment (ptersilie) [inline doc/src/lexing.md:11]: Typos `A a`

Comment (ltratt) [reply doc/src/lexing.md]: Fixed in https://github.com/softdevteam/grmtools/pull/199/commits/c874d4fc7413eb9b94f3145d1b81fda32ef2168a.

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**